src/resources/MachineLearning/UserActionHistoryConfiguration/UserActionHistoryConfiguration.ts

```diff
@@ -0,0 +1,18 @@
+import API from '../../../APICore';
+import Resource from '../../Resource';
+
+export default class UserActionHistoryConfiguration extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/machinelearning/configuration/useractionhistory`;
+
+    create() {
```

Comment:
instead of `create` and `delete` would it make more sense to have `enable` and `disable`?

Reply:
ML called the calls like this, so I kept their names 
![image](https://user-images.githubusercontent.com/2488155/147000870-435edeaa-b8df-473e-b281-435bd3e0a459.png)